### ISSUE TITLE
Server freezes during load after latest update

### ISSUE THREAD

**Issue by ZyniX69** [MoriyaShiine/bewitchment#51]
Updated my lan server from bewitchment-1.16.5-4 to bewitchment-1.16.5-5.jar and I couldn't get the server to finish loading. It works fine when I revert to the last bewitchment update. This is point it freezes at and there was no crashlog.

https://pastebin.com/pFci4Urt

**Comment by Aelpecyem** [comment]
your log is barely legible, can you please put your log in a separate paste i.e. by using https://pastebin.com/ ?

**Comment by ZyniX69** [comment]
Here is the full log from startup to the freeze. https://pastebin.com/abS972Ff

**Comment by MoriyaShiine** [comment]
I can't reproduce this, can you isolate what mod causes this

**Comment by CannibalKon** [comment]
I am having a similar issue. Singleplayer, not server of any kind. When I go to create a world, it crashes before finishing and spews out an error. Didn't have this issue before the update with this entire list, only began once the update happened.

https://pastebin.com/aSKGQ376

**Comment by ZyniX69** [comment]
I have done a bit of testing making a copy of my server and removing mods one at a time with no luck unless I removed 90% of the mods. I've also created a new server from scratch and added bewitchment as well as mods one by one after it could run with bewitchment+dependents alone. 

On the fresh server I've been using mod lists as small as 20 mods and bewitchment is causing other mods to conflict with each other. Usually mods that I thought were the culprit because the server would run after removed ended up working as long as a different mod was removed instead. I've tested this with quite a few mod combinations. Also if I run the server with only bewitchment and both of the "conflicting" mods the server works fine. It only appears to freeze over 20ish mods.

Now I can take all 150+ mods and run fine with the last version of bewitchment so that is what my wife and I will do in the meantime. Sorry if this doesn't help but I've spent about 3 hours trying to figure out the problem and I am no expert.
